### ISSUE TITLE
A question about your loss_coteaching function

### ISSUE THREAD

**Issue by nihaomiao** [bhanML/Co-teaching#4]
Hello, I find that in your class "loss_coteaching", the parameters you passed are y_1, y_2, which are already the log_softmax results. But you then used the [cross_entropy](https://pytorch.org/docs/stable/nn.html#torch.nn.functional.cross_entropy), which has combined the log_softmax and nll_loss. This will use log_softmax twice. I am not sure whether I am wrong. Or do this problem not occur in your pytorch version?

**Comment by bbdamodaran** [comment]
I also observed the same issue, but I tested using log_softmax only once. I did not observe the difference in the results. Please let me know if you observe different results. One more issue is that in return function of loss, the loss value is normalized twice

**Comment by nihaomiao** [comment]
OK~

**Comment by guixianjin** [comment]
I don't observe that y_1, y_2 are already the log_softmax results, have the code has been changed and fixed?  @nihaomiao @bbdamodaran

**Comment by henrypickler** [comment]
This was fixed in 4d32d1c. Note that even if the results were already `log_softmax` transformed this would not change the results since applying `log_softmax` twice is the same as applying it only once. It also does not change the ordering for selecting samples for each model meaning it didn't had any effect other than a bit of extra computation.
